source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -552,22 +604,70 @@ namespace AdaptiveCardsSharedModelUnitTest
             MarkDownParser parser("5>3");
             Assert::AreEqual<std::string>("<p>5&gt;3</p>", parser.TransformToHtml());
         }
+
         TEST_METHOD(EscapeHtmlCharactersTest_LessThanTest)
         {
             MarkDownParser parser("3<5");
             Assert::AreEqual<std::string>("<p>3&lt;5</p>", parser.TransformToHtml());
         }
+
         TEST_METHOD(EscapeHtmlCharactersTest_QuotationTest)
         {
             MarkDownParser parser("\"Hello World!\"");
             Assert::AreEqual<std::string>("<p>&quot;Hello World!&quot;</p>", parser.TransformToHtml());
         }
+
         TEST_METHOD(EscapeHtmlCharactersTest_AmpersandTest)
         {
             MarkDownParser parser("Green Eggs & Ham");
             Assert::AreEqual<std::string>("<p>Green Eggs &amp; Ham</p>", parser.TransformToHtml());
         }
 
+        TEST_METHOD(NonLatinCharacters_NoMarkdown)
+        {
+            MarkDownParser parser("以前の製品のリンクで検索");
+            Assert::AreEqual<std::string>("<p>以前の製品のリンクで検索</p>", parser.TransformToHtml());
+        }
+
+        TEST_METHOD(NonLatinCharacters_Bold)
+        {
+            MarkDownParser parser("**以前の製品のリンクで検索**");
+            Assert::AreEqual<std::string>("<p><strong>以前の製品のリンクで検索</strong></p>", parser.TransformToHtml());
+        }
+
+        TEST_METHOD(NonLatinCharacters_BoldSentence)
+        {
+            MarkDownParser parser("How about **以前の製品のリンクで検索**");
+            Assert::AreEqual<std::string>("<p>How about <strong>以前の製品のリンクで検索</strong></p>", parser.TransformToHtml());
+        }
+
+        TEST_METHOD(NonLatinCharacters_BoldSentenceNoWhitespace)
+        {
+            MarkDownParser parser("How about**以前の製品のリンクで検索**");
```

Comment:
would be nice to have (a) test(s) that have non-Latin characters appear before emphasis opening delimiters and / or after emphasis closing delimiters.